Makefile

```diff
@@ -20,12 +20,12 @@ uninstall: $(MAKEFILE_COQ)
 $(MAKEFILE_COQ): _CoqProject
 	coq_makefile -f $< -o $@
 
-COMPATFILES:=plugin/compat.ml
+COMPATFILES:=plugin/coqsimpleio.mlg
 
 compat: $(COMPATFILES)
 
-$(COMPATFILES): compat.pl
-	$(V)perl -- compat.pl $(COMPATFILES)
+%: %.cppo
+	$(V)cppo -V COQ:$(word 1, $(shell coqc -print-version)) -n -o $@ $^
```

Comment:
Is there a reason for setting `-n`? It makes sense for Coq files, but `-s` seems nicer for OCaml files.

Reply:
Does cppo print the right file without `-n`? If yes I don't mind unsetting this flag.

Reply:
Ah, nevermind, it goes through coqpp which doesn't understand line directives...